Update Tagging Best Practices index.html

wfreeman-aph opened the pull request:
The changes in this commit address the following issues with the Tagging Best Practices document:

- 149 (by adding James' explanation to the note at the beginning)
- 147 (by making the changes recommended)
- 146 (by fixing anywhere two class attributes were used incorrectly)
- 145 (by changing tn to transcribers-note and changing linenumber to linenum)

clapierre reviewed (approved):
Changes look great. Love the more detailed class names to help in readability.

mattgarrish commented:
Just a side note, but if you put a # in front of the issue number, github will make a link to the issue. I modified the first message to add the links. If you want the issues automatically closed when you merged, add "Fixes " in front of each issue number.

egli reviewed (approved):
Excellent

wfreeman-aph commented:
> Just a side note, but if you put a # in front of the issue number, github will make a link to the issue. I modified the first message to add the links. If you want the issues automatically closed when you merge, add "Fixes " in front of each issue number.

Thanks @mattgarrish! I knew about the # thing and have used it in other places and should have realized to use it here. I'm glad you caught it and thanks for telling me about the "fixes" feature. I added that word to the pull request and then also @jrbowden since some of the tickets that will close are his.

Thanks for reviewing @egli!
